### PR TITLE
DE7070-handle-zero-state

### PR DESCRIPTION
**Pull request by J-Byron**
- Handles zero states for song & album sections
- Handles zero state for logo-block

** note: "Back to album" button doesn't work for songs without an album
## Test Cases

Albums
- [Missing youtube link ](https://deploy-preview-22--crdsmusic-int.netlify.com/)
- [Missing songs](https://deploy-preview-22--crdsmusic-int.netlify.com/music/you-are-able)

Songs
- [Missing lyrics and album](https://deploy-preview-22--crdsmusic-int.netlify.com/music/faithful-you-are-able)
- [Missing video](https://deploy-preview-22--crdsmusic-int.netlify.com/music/whatever-pleases-you/anybody-else)
- [Missing album](https://deploy-preview-22--crdsmusic-int.netlify.com/music/let-me)

### CODE REVIEW

File: _includes/_logo-block.html [hunk 1]
```diff
@@ -1,13 +1,19 @@
 <div class="logo-block {{ include.class }}">
   <div class="logo-block-content">
-    <a href="{{ include.content.spotify_url }}" class="logo-block-logo" target="_blank">
+    {% if include.links.spotify_url %}
+    <a href="{{ include.links.spotify_url }}" class="logo-block-logo" target="_blank">
       {% include svg/_spotify-logo.svg %}
     </a>
-    <a href="{{ include.content.youtube_url }}" class="logo-block-logo" target="_blank">
+    {% endif %}
```
Comment (seancdavis) [inline _includes/_logo-block.html]: More indentation things.

Comment (seancdavis) [reply _includes/_logo-block.html]: Check this throughout. Consider getting the HTML Liquid plugin for VS Code and using a local (workspace) association for html files to use the HTML Liquid language.

File: _layouts/song.html [hunk 2]
```diff
@@ -53,7 +61,10 @@ <h4 class="page-section-title">album</h4>
...
     </div>
+    {% endif %}
   </div>
-</div>
+  {% endif %}
 
-{% include _logo-block.html links=page %}
+  {% if page.youtube_url or page.spotify_url or page.apple_music_url %}
+  {% include _logo-block.html links=page %}
+  {% endif %}
```
Comment (seancdavis) [inline _layouts/song.html]: Found a regression:

Screen Shot 2019-08-12 at 10 14 34 AM

**Comment by seancdavis** [comment]
I'd also like to see an example where I can test all zero states and another where I can see a song with every attribute filled out. Please send an example of both.
